sympy/physics/optics/tests/test_utils.py

```diff
@@ -178,3 +178,26 @@ def test_transverse_magnification():
     si, so = symbols('si, so')
     assert transverse_magnification(si, so) == -si/so
     assert transverse_magnification(30, 15) == -2
+
+def test_lens_makers_formula_thick_lens():
```

Comment:
Should be two blank lines between `test_` functions (or other top-level) statements. Can you fix that throughout the file which is currently inconsistent.

Reply:
Yes I'll do that